packages/node/src/configure/project.model.ts

```diff
@@ -1,13 +1,21 @@
 // Copyright 2020-2021 OnFinality Limited authors & contributors
 // SPDX-License-Identifier: Apache-2.0
 
-import { ProjectManifest, SubqlDataSource } from '@subql/common';
+import {
+  loadProjectManifest,
+  ProjectManifest,
+  SubqlDataSource,
+} from '@subql/common';
+import { prepareProjectDir } from '../utils/project';
 
 export class SubqueryProject implements ProjectManifest {
-  static create(source: ProjectManifest, path: string): SubqueryProject {
+  static async create(path: string): Promise<SubqueryProject> {
+    const projectPath = await prepareProjectDir(path);
+    console.log(projectPath);
```

Comment:
Is this necessary output? remove if it is only for debugging

Reply:
removed